### ISSUE TITLE
Panel .open() and .close() rely on a single global busy state

### ISSUE THREAD

**Issue by amtgard** [quark-dev/Phonon-Framework#64]
Meaning that independent panels cannot be controlled simultaneously without relying on cumbersome calls to setTimeout();

**Comment by LucasBurg** [comment]
Hy, do you write an examplo, please ?

**Comment by bdedardel** [comment]
Hi,

I have the same issue.

### html

```
<div class="side-panel side-panel-left" data-expose-aside="left" data-disable="right" data-page="myfirstpage" id="sidepanel">
	<header class="header-bar">
		<button class="btn pull-right icon icon-close show-for-phone-only" data-side-panel-close="true"></button>                
	</header>
	<div class="content">
		<ul class="list">                                       		
			<li><a class="padded-list" onclick="openPanel();">Open panel</a></li>  
		</ul>
	</div>
</div>
<div id="mypanel" class="panel-full">
		<header class="header-bar">
			<a class="btn icon icon-close pull-right" href="#" data-panel-close="true"></a>
			<h1 class="title">My panel</h1>
		</header>
		<div class="content">
			<p class="padded-full">
				<a href="#" id="closeButton" onclick="closePanel()">Close panel programmaticaly</a>
			</p>
		</div>
</div>
```

### js

```
function openPanel() {
	// ...
	phonon.panel('#mypanel').open();
}

function closePanel() {
	// ...
	phonon.panel('#mypanel').close();
}
```

Open/close panel works fine many times on linux/firefox.
Open/close panel works **just on time** on android/cordova/crosswalk.

Regards,
Benjamin

**Comment by bdedardel** [comment]
Hi, 

It works fine with the last release.

Thanks.

**Comment by amtgard** [comment]
Outstanding! I'll check it out.

Thank you.

**Comment by Qathom** [comment]
Thank your for reporting this issue.
I just removed the global state, now each panel has its own state.
The patch will be available in the **release 1.2.0** (coming asap).
